### PR TITLE
updated powersploit-payload

### PR DESCRIPTION
**Pull request by thel3l**
This updates the powersploit-payload file to a version available at [Empire](https://github.com/PowerShellEmpire/Empire/blob/master/data/module_source/code_execution/Invoke-Shellcode.ps1)

Fixes half the issue at: https://github.com/offensive-security/kali-nethunter/projects/8

@binkybear this fixes the outdated script.

### CODE REVIEW

**Comment by binkybear** [comment]
Here is the issue I was referencing: https://github.com/offensive-security/nethunter-app/issues/114

One of the things I need to make sure works is the HID attack with this updated powershell script.  

Right now in the app we launch:
hid-all.sh (pass arguments to hid-all)

> hid-all.py: https://github.com/offensive-security/nethunter-utils/blob/master/usr/bin/hid-all.py
> 
> > rev-tcp: https://github.com/offensive-security/nethunter-app/blob/master/assets/nh_files/configs/rev-tcp

The app actually adds a string to the end of /var/www/html/powersploit-url (file you are submitting):
https://github.com/offensive-security/nethunter-app/blob/master/src/com/offsec/nethunter/HidFragment.java#L390

Long story short, just need to test to make sure this still works or if I need to update app.

**Comment by thel3l** [comment]
Understood. Go ahead and close this pull request if it doesn't need to be updated @binkybear

**Comment by binkybear** [comment]
I think it needs to be updated.  I just need to make sure it works with existing app or if I need to change.  The payload is a bit dated. (Actually I can't accept pulls in this repo either)

**Comment by thel3l** [comment]
Aah lel. Didn't see that either. 

Okay. I'll take a look at something else in the project list in the meantime.

**Comment by jmingov** [comment]
Closing. Was solved in https://github.com/offensive-security/nethunter-app/pull/145
